src/main/java/DoneCommand.java

```diff
@@ -0,0 +1,28 @@
+import java.io.IOException;
+
+public class DoneCommand extends Command {
+    private int doIndex;
+
+    public DoneCommand(int doIndex) {
+        this.doIndex = doIndex;
+    }
+
+    @Override
+    public boolean isExit() {
+        return false;
+    }
+
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
+        Task chosen_Task = tasks.getTask(doIndex - 1);
```

Comment:
I noticed a number of variables where you used a combination of snake case and camel case. I think you should change it to strictly camel casing for this module.